lua/nvim-autopairs/completion/cmp.lua

```diff
@@ -4,9 +4,16 @@ local cmp = require('cmp')
 
 local M = {}
 M.setup = function(opt)
-    opt = opt or { map_cr = true, map_complete = true, auto_select = true }
+    opt = opt or { map_cr = true, map_complete = true, auto_select = true, map_char = {all = '('} }
+    if not opt.map_char then
+        opt.map_char = {all = '('}
+    end
+    if opt.map_char["all"] == nil then
+        opt.map_char["all"] = '('
+    end
```

Comment:
we don't need 'all' type you can set the default later
it should be map_char ={ tex= ' ' }

Reply:
What about this:
```lua
opt = opt or { map_cr = true, map_complete = true, auto_select = true, map_char = {all = '('} }
if not opt.map_char then opt.map_char = {} end
```
Then, on `completion_done`/`on_confirm_done`, it will be checked like:
```lua
local char = map_char[filetype] or map_char["all"] or '('
if char == '' then return end
```